src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -206,6 +206,18 @@ public void removeWriteOnlyProperties(final JSONObject resourceModel) {
             new PublicJSONPointer(writeOnlyProperty.replaceFirst("^/properties", "")), resourceModel));
     }
 
+    public boolean compareCreateOnlyProperties(final JSONObject previousResourceModel,
```

Comment:
should this go into the schema class? seems like it more just utilizes the schema's properties. are we using this in multiple places? if not can we just include the logic where  we are doing the comparisons?

Reply:
We will be using this for all plugins. This was the reason I added it to schema package